src/main/java/com/shopify/minishopify/controllers/ShopController.java

```diff
@@ -102,4 +102,20 @@ public List findDupes(List<Shop> l1, List<Shop> l2){
         }
         return l1;
     }
+
+    @DeleteMapping("/delete/{id}")
+    public Shop deleteShop(@PathVariable int id) {
+
+        Optional<Shop> shopQuery = shopRepository.findById(id);
+
+        if (shopQuery.isPresent()) {
+
+            Shop shopToDelete = shopQuery.get();
+            shopRepository.delete(shopToDelete);
```

Comment:
Will this delete the user if the user has no other stores? I had issues with the Product deleting the Shop if it was the last product in the shop when deleted.

Reply:
I was able to solve it for my case, just wondering if it impacted yours.